IBPSA/Fluid/Geothermal/Aquifer/MultiWell.mo

```diff
@@ -0,0 +1,428 @@
+<p>
+This model simulates aquifer thermal energy storage, using one or multiple instances of cold and hot wells.
+</p>
+<p>
+To calculate aquifer temperature at different locations over time, the model applies
+theoretical principles of water flow and heat transfer phenomena. The model is based on
+the partial differential equation (PDE) for 1D conductive-convective transient
+radial heat transport in porous media
+</p>
+<p align=\"center\" style=\"font-style:italic;\">
+&rho; c (&part; T(r,t) &frasl; &part;t) =
+k (&part;&sup2; T(r,t) &frasl; &part;r&sup2;) - &rho;<sub>w</sub> c<sub>w</sub> u(&part; T(r,t) &frasl; &part;t),
+</p>
+<p>
+where
+<i>&rho;</i>
+is the mass density,
+<i>c</i>
+is the specific heat capacity per unit mass,
+<i>T</i>
+is the temperature at location <i>r</i> and time <i>t</i>,
+<i>u</i> is water velocity and
+<i>k</i> is the heat conductivity. The subscript <i>w</i> indicates water.
+The first term on the right hand side of the equation describes the effect of conduction, while
+the second term describes the fluid flow.
+</p>
+<h4>Spatial discretization</h4>
+<p>
+To discretize the conductive-convective equation, the domain is divided into a series
+of thermal capacitances and thermal resistances along the radial direction. The
+implementation uses an array of
+<a href=\"modelica://Modelica.Thermal.HeatTransfer.Components.HeatCapacitor\">Modelica.Thermal.HeatTransfer.Components.HeatCapacitor</a>
+and <a href=\"modelica://Modelica.Thermal.HeatTransfer.Components.ThermalResistor\">Modelica.Thermal.HeatTransfer.Components.ThermalResistor</a>
+Fluid flow was modelled by adding a series of fluid volumes, which are connected
+to the thermal capacitances via heat ports. The fluid stream was developed using
+the model <a href=\"modelica://IBPSA.Fluid.MixingVolumes.MixingVolume\">IBPSA.Fluid.MixingVolumes.MixingVolume</a>.
+The geometric representation of the model is illustrated in the figure below.
+</p>
+<p align=\"center\">
+<img  alt=\"image\" src=\"modelica://IBPSA/Resources/Images/Fluid/Geothermal/Aquifer/Geometry.png\" width=\"800\">
+</p>
+<h4>Typical use and important parameters</h4>
+<p>
+By default, the component consists of a cold well and a warm well. The number of wells can be increased by modifing the parameters <code>nCoo</code> and <code>nHot</code>.
+The effect is a proportional increase of thermal capacity.
+</p>
+<p>
+To ensure conservation of energy, the two wells are connected via fluid ports. To avoid thermal interferences, make sure that the aquifer domain radius <code>r_max</code> is large enough for your specific use case.
+</p>
+<p>
+Circulation pumps are included in the model and they can be controlled by acting on the input connector. The input must vary between [1,-1]. A positive value will circulate water
+clockwise (extraction from the cold well and injection into the warm well). A negative value will circulate water anticlockwise (extraction from the warm well and injection into the cold well).
```

Comment:
Here you should also explain the design pressure drops and how they are used to size the pump.

Reply:
I think that the new illustration should be able to explain the rationale behind the selection of the design pressure drops.